TeaLeaf/src/com/tealeaf/TeaLeaf.java

```diff
@@ -308,7 +308,7 @@ public void onCreate(Bundle savedInstanceState) {
 		int orientation = getRequestedOrientation();
 
 		// gets real screen dimensions without nav bars on recent API versions
-		if (isFullScreen && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT) {
+		if (isFullScreen && Build.VERSION.SDK_INT >= Build.VERSION_CODES.KITKAT && !isNotchedDevice()) {
```

Comment:
we are using deprecated API for notched devices.
we need to use display.getSize() for the devices wit ha notch